SOURCE/header_MDOODZ.h

```diff
@@ -203,7 +203,7 @@ struct _grid {
     double *VxVz, *VzVx;
     int    *P2N, *P2C;
     int    *kvx, *lvx, *kvz, *lvz, *kp, *lp, *kn, *ln;
-    double **phase_perc_n, **phase_perc_s;
+    double **phase_perc_n, **phase_perc_s,  **phase_visc_n, **phase_visc_s;
```

Comment:
What's the difference between phase_visc_n and phase_visc_s?
How does cell centers and vertices reflected in that?

Reply:
TODO: need to be refactored in the future.

Reply:
I've already changed to `phase_eta_` to be consistent with other viscosity arrays